server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileReset.java

```diff
@@ -110,7 +114,7 @@ public void resetLanguage(String language) {
   BulkChangeResult reset(QProfileName profileName, Collection<RuleActivation> activations) {
     DbSession dbSession = db.openSession(false);
     try {
-      QualityProfileDto profile = factory.getOrCreate(dbSession, profileName);
+      QualityProfileDto profile = factory.getOrCreate(dbSession, qProfileWsSupport.getDefaultOrganization(dbSession), profileName);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "getDefaultOrganization"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)